src/foundry/foundry.js/clientDocuments/actor.d.ts

```diff
@@ -25,10 +29,191 @@ declare global {
+     * @param document - Return the Document instance rather than the PlaceableObject (default: `false`)
+     * @returns An array of Token instances in the current Scene which reference this Actor.
+     */
+    getActiveTokens(linked?: boolean, document?: boolean): Token[];
+
+    /**
+     * Prepare a data object which defines the data schema used by dice roll commands against this Actor
+     */
+    getRollData(): this['data']['data'];
+
+    /**
+     * Create a new TokenData object which can be used to create a Token representation of the Actor.
+     * @param data - Additional data, such as x, y, rotation, etc. for the created token data
+     * @returns The created TokenData instance
+     * TODO: TokenData does not yet exist
+     */
+    getTokenData(data: object): Promise<object>;
+
+    /**
+     * Get an Array of Token images which could represent this Actor
+     */
+    getTokenImages(): Promise<string[]>;
+
+    /**
+     * Handle how changes to a Token attribute bar are applied to the Actor.
+     * This allows for game systems to override this behavior and deploy special logic.
+     * @param attribute - The attribute path
+     * @param value - The target attribute value
+     * @param isDelta - Whether the number represents a relative change (true) or an absolute change (false) (default: `false`)
+     * @param isBar - Whether the new value is part of an attribute bar, or just a direct value (default: `true`)
+     * @returns  The updated Actor document
+     */
+    modifyTokenAttribute(
+      attribute: string,
+      value: number,
+      isDelta: boolean,
+      isBar: boolean
+    ): Promise<foundry.documents.BaseActor>;
+
+    /**
+     * Roll initiative for all Combatants in the currently active Combat encounter which are associated with this Actor.
+     * If viewing a full Actor entity, all Tokens which map to that actor will be targeted for initiative rolls.
+     * If viewing a synthetic Token actor, only that particular Token will be targeted for an initiative roll.
+     *
+     * @param options - Configuration for how initiative for this Actor is rolled.
+     * @returns A promise which resolves to the Combat entity once rolls are complete.
+     */
+    rollInitiative(options?: ActorRollInitiativeOptions): Promise<void>;
+
+    /**
+     * Refresh the display of active Tokens for this Actor if ActiveEffects were changed
+     */
+    protected _refreshTokens(): void;
+
+    /**
+     * You are referencing Actor#_data which has been deprecated in favor of Actor#data#_source. Support for this reference will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    get _data(): this['data']['_source'];
+
+    /**
+     * You are referencing Actor#getOwnedItem(itemId) which is deprecated in favor of Actor#items#get(itemId). Support will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    getOwnedItem(itemId: string): Item | null;
```

Comment:
should probably use the configured `Item` class. Also it will _not_ return `null`. If the `itemId` does not exist, it will return `undefined`. I suppose you could use something like `ReturnType<this['items']['get']>` instead of writing the type manually.